src/Compilers/VisualBasic/Portable/Compilation/MethodCompiler.vb

```diff
@@ -33,6 +33,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         ' It normally happens during actual compile, but also happens when getting emit diagnostics for 
         ' testing purposes.
         Private ReadOnly _doEmitPhase As Boolean
+        Private ReadOnly _doLoweringPhase As Boolean
```

Comment:
>_doLoweringPhase [](start = 25, length = 16)

nit: I'd consider leaving a comment why/when this is useful. To get used assemblies we need to run lowering phase, but for regular diagnostic information we don't.